shared/collapse-generated-files.sh

```diff
@@ -74,8 +74,11 @@ fi
 
 # The generated file was not the same across all platforms, so
 # put the different versions in __$PLATFORM__ subfolders.
-for PLATFORM in linux64 macosx64 win64 android-armv7; do
+for PLATFORM in linux64 macosx64 win64 android-armv7 ios; do
     if [ ! -f "generated-$PLATFORM/$GENERATED_FILE" ]; then
+	if [ $PLATFORM = ios ]; then
```

Comment:
This change seems like it merits a comment explaining what's going on.

Reply:
Note that practically speaking, it's not useful because ios is last, but in case it stops being last...